arbnode/resourcemanager/resource_management.go

```diff
@@ -46,7 +46,7 @@ func Init(conf *Config) error {
 		var c limitChecker
 		c, err := newCgroupsMemoryLimitCheckerIfSupported(limit)
 		if errors.Is(err, errNotSupported) {
-			log.Error("No method for determining memory usage and limits was discovered, disabled memory limit RPC throttling")
+			log.Error("no method for determining memory usage and limits was discovered, disabled memory limit RPC throttling")
```

Comment:
nit: This was correct previously. It should be uppercase. You can remember it with: if it's displaying the text immediately, that should be uppercase, if it's returned then lowercase [0].

Same below at ln 120: `log.Error("error checking memory limit", "err", err, "checker", s.c)`. Moreover at that line  `s.c` is an interface (limitChecker) so logging that will probably just log pointers to those two methods (String() and isLimitExceeded()) which wouldn't be very useful for debugging. I would log `s.c.String()` instead).

https://google.github.io/styleguide/go/decisions#error-strings